thoth/python/package_version.py

```diff
@@ -393,13 +393,31 @@ def to_tuple(self) -> Tuple[str, str, Optional[str]]:
 
         return self._package_tuple
 
+    def to_strict_tuple(self) -> Tuple[str, str, str]:
+        """Return a tuple representing this Python package, packages with index assigned."""
+        assert self.index is not None
+
+        if self._package_tuple:
```

Comment:
```suggestion
        if self._package_tuple is None:
```
I think this should be a check for None